test/AnalyticsE2ETest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+phpinfo();
+
+require_once(dirname(__FILE__) . "/../lib/Segment.php");
+
+class AxiosClient
+{
+  private $BaseAddress = null;
+  private $Timeout = 3000;
+  private $Authorization = "";
+  private $RetryCount = 1;
+
+  public function __construct($baseAddress, $timeout, $authorization) {
+    $this->BaseAddress = $baseAddress ? $baseAddress : "https://api.runscope.com";
+    if ($timeout)
+      $this->Timeout = $timeout;
+    if ($authorization)
+      $this->Authorization = $authorization;
+  }
+
+  public function setRetryCount($count) {
+    $this->RetryCount = $count > 0 ? $count : 1;
+  }
+
+  public function get($url) {
+    for ($i = 0; $i < $this->RetryCount; $i++) {
+      // open connection
+      $ch = curl_init();
+
+      // Set timeout
+      curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, $this->Timeout);
+      curl_setopt($ch, CURLOPT_TIMEOUT, $this->Timeout);
+
+      // Set authorization
+      if ($this->Authorization) {
+        $header = array();
+        $header[] = 'Authorization: Bearer ' . $this->Authorization;
+
+        curl_setopt($ch, CURLOPT_HTTPHEADER, $header);
+      }
+
+      // Set url
+      curl_setopt($ch, CURLOPT_URL, $this->BaseAddress . "/" . $url);
+      curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
+
+      $output = curl_exec($ch);
+      $response = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+      curl_close($ch);
+
+      if ((int)$response == 200)
+        return $output;
+
+      usleep(3);
+    }
+
+    return NULL;
+  }
+}
+
+class AnalyticsE2ETest extends PHPUnit_Framework_TestCase {
+
+  private static $WRITE_KEY = "Cum1GHo14CegfLd83gVxsbc5Sb6jK8SN";
+  private static $RUNSCOPE_BUCKET = "pwb8mcmfks0f";
+  private $id = "";
+
+  private static function messageId(){
+    return sprintf("%04x%04x%04x%04x%04x%04x%04x%04x"
+      , mt_rand(0, 0xffff)
+      , mt_rand(0, 0xffff)
+      , mt_rand(0, 0xffff)
+      , mt_rand(0, 0x0fff) | 0x4000
+      , mt_rand(0, 0x3fff) | 0x8000
+      , mt_rand(0, 0xffff)
+      , mt_rand(0, 0xffff)
+      , mt_rand(0, 0xffff));
+  }
+
+  function setUp() {
+    date_default_timezone_set("UTC");
+
+    $this->id = self::messageId();
+
+    // Segment Write Key for https://segment.com/segment-libraries/sources/analytics_node_e2e_test/overview.
```

Comment:
Let's change the link to https://app.segment.com/segment-libraries/sources/analytics_php_e2e_test/overview

Reply:
hey @kevingilliard, could you update the link?

Reply:
Hi @f2prateek 
I updated this comment.